src/smc-util/message.coffee

```diff
@@ -572,11 +572,48 @@ message
 
 # Unlink a passport auth for this account.
 # client --> hub
-API message
+API message2
     event    : 'unlink_passport'
-    strategy : required
-    id       : required
+    fields:
+        strategy:
+            init  : required
+            desc  : 'passport strategy'
+        id:
+            init  : required
+            desc  : 'numeric id for user and passport strategy'
+    desc:"""
+Unlink a passport auth for the account.
+
+Strategy is one of: `facebook`, `github`, `google`, `twitter`.
```

Comment:
Yep, and there are way more than 4 already implemented.  I've been waiting for somebody to requests a fifth... Never happens.